src/components/IspSelect/IspSelect.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../assets/variables";
+
+.IspSelect {
```

Comment:
So the other components have been following the convention of converting to dashed case for class name (e.g., `.isp-select`) -- I'm happy with keeping component root class names in the same case as the component name itself as you've done here, we'll just have to update the others. Thoughts?